packages/styles/scss/patterns/sub-patterns/content-group/_content-group.scss

```diff
@@ -22,31 +20,40 @@
         margin-bottom: $carbon--layout-05;
       }
     }
-
     @include carbon--breakpoint(lg) {
       padding-top: $carbon--layout-05;
       padding-bottom: $carbon--layout-07;
-
       &__title {
         margin-bottom: $carbon--layout-03;
       }
     }
 
     &__title {
-      padding-left: $carbon--layout-01;
       padding-right: $carbon--layout-01;
       max-width: 90%;
 
       @include carbon--type-style('expressive-heading-04', true);
     }
 
     &__children {
-      padding-left: 0;
-      padding-right: 0;
+      margin-bottom: $carbon--layout-03;
+    }
+
+    &__CTA {
+      @include carbon--breakpoint('sm') {
+        @include carbon--make-col(1, 2);
+      }
+      @include carbon--breakpoint('md') {
+        @include carbon--make-col(2, 4);
```

Comment:
small tweak here. The CTA is 4 of 8. Same value here, but at the md breakpoint the grid is now 8.